src/theme/colors.ts

```diff
@@ -44,4 +45,16 @@ export const colors = {
 
   // WARNING: This color is an abomination and should not be used! But we're using it anyway.
   controversialBlue: '#168FFF',
+
+  // Used specifically for the sarcophagus table
+  // Note that there are duplicate colors through the app because the colors in the designs have
+  // changed over time. These colors need to be standardized.
+  // TODO: Get with design to standardize some of these colors.
+  blueBg: '#102233',
+  red: '#F74141',
+  redBg: '#2f1414',
+  green: '#17CB49',
+  greenBg: '#0b2613',
+  yellow: '#FF9F2D',
+  yellowBg: '#342615',
```

Comment:
I don't like how these colors are defined. We are having discussions about it on discord. We need to go through the app and make sure the app color palette matches the figma color palette as much as possible.